client/src/routes/home/Home.js

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
 import withStyles from 'isomorphic-style-loader/withStyles';
+import CarbonWidget from '../carbonsensor/CarbonWidget/carbon-widget';
 import s from './Dashboard.css';
```

Comment:
Is this called Home or Dashboard? Please remove the folder and all files to be called `dashboard` or `Dashboard`